doc/compodoc_sources/concepts/permissions.md

```diff
@@ -111,6 +111,70 @@ The `admin_app` role simpy allows user with this role to do everything, without
 To learn more about how to define rules, have a look at
 the [CASL documentation](https://casl.js.org/v5/en/guide/define-rules#rules).
 
+## Defining permissions
+The permissions are also stored in a CouchDB database (default database name `app`).
+This can be the same database used for normal application data managed by users.
+
+The structure of the permission document is as follows:
+```json
+{
+  "_id": "Config:Permissions",
+  "data": {
+    "public": [
+      { "subject": "User", "action":  "create"}
+    ],
+    "default": [
+      { "subject":  "Config", "action": "read" }
+    ],
+    "role_1": [
+      { "subject":  "all", "action":  "manage"},
+      ...
+    ],
+    "role_2": [],
+    ...
+  }
+}
+```
+Important is the exact `_id` as this is how the backend can find this document and that the rules config has the correct structure.
```

Comment:
This is duplicating part of the above "Permission Structure" section?

Reply:
removed the duplicating part